Add onnx ArgMin node

tye-singwa opened the pull request:
## Pull Request Template

### Checklist

- [x] Confirmed that `cargo run-checks` command has been executed.
- [x] Made sure the book is up to date with changes in this PR.

### Related Issues/PRs

#1714

### Changes

Added  ArgMin onnx node same way as ArgMax is implemented right now

### Testing

Code covered with tests, including sample `argmin.onnx`

laggui reviewed (commented):
LGTM!

Just missing the change in [SUPPORTED-ONNX-OPS.md](https://github.com/tracel-ai/burn/blob/main/crates/burn-import/SUPPORTED-ONNX-OPS.md) 🙂

tye-singwa commented:
Thanks for checking, just rebased, added [SUPPORTED-ONNX-OPS.md](https://github.com/tracel-ai/burn/blob/main/crates/burn-import/SUPPORTED-ONNX-OPS.md), missed it
